internal/crawler/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	HTTPClientMaxRetry     int           `conf:"default:2,short,m,env:HTTP_CLIENT_MAX_RETRY"`
 	HTTPClientMaxRetryWait time.Duration `conf:"default:30s,short:w,env:HTTP_CLIENT_MAX_RETRY_WAIT"`
 	HTTPClientMinRetryWait time.Duration `conf:"default:5s,short:n,env:HTTP_CLIENT_MIN_RETRY_WAIT"`
+	numberOfWorkers        int           `conf:"default:20,short:wr,env:NUMBER_OF_WORKERS"`
```

Comment:
Should be upper case like the rest. Also two letter shorthands feel weird in POSIX style CLI configs.
```suggestion
	NumberOfWorkers        int           `conf:"default:20,short:c,env:NUMBER_OF_WORKERS"`
```